webpack/assets/javascripts/react_app/components/SubmitCancelButtons/SubmitCancelButtons.test.js

```diff
@@ -0,0 +1,203 @@
+import toJson from 'enzyme-to-json';
+import { shallow } from 'enzyme';
+import React from 'react';
+import { SubmitCancelContainer } from './index';
+import Actions from '../common/forms/Actions';
+
+const data = { cancelPath: 'test' };
+const onMountButtons = jest.fn();
+
+describe('submit_or_cancel_buttons', () => {
```

Comment:
We found out it is much easier to test **unconnected** components against snapshots.
considre exporting both connected and unconnected to make it more reusable and testable.